Serenity November (#1061)
akeneo/pim-helpcenter: content/updates/2021-11/3-APIendpoint.md

---
pim_announcement_img:
pim_announcement_alt_img:
pim_announcement_audience:
- EE
- CE
---

# New API endpoint to delete product models
::: meta-data type="New" features="Productivity, automation" available="Late-November" in="GE,EE" link-to-doc="https://api.akeneo.com/api-reference.html#delete_product_models__code_"

> We are happy to introduce a new API endpoint DELETE /api/rest/v1/product-models/{code} to quickly and easily delete a given product model with its sub-product models and variant products. This new endpoint, which completes the CRUD operations available through our REST API, allows you to manage with more flexibility and agility product family changes in your PIM. It also eases data synchronization between your sources of data (ERP) that leverage our API and your PIM.

Reviewer: ```suggestion
Introducing a new API endpoint DELETE /api/rest/v1/product-models/{code}. Quickly and easily delete a given product model with its sub-product models and variant products. This new endpoint, which completes the CRUD operations available through our REST API, allows you to manage product family changes in your PIM with more flexibility and agility. It also eases data synchronization between your sources of data (like your ERP) that leverage our API and your PIM.
```
